browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -107,17 +107,6 @@ class ToggleButton : public ToolbarButton {
     return l10n_util::GetStringUTF16(IDS_VERTICAL_TABS_EXPAND);
   }
 
-#if DCHECK_IS_ON()
-  void OnBoundsChanged(const gfx::Rect& previous_bounds) override {
-    if (region_view_->is_animating()) {
-      return;
-    }
-
-    CHECK_EQ(width(), GetIconWidth());
```

Comment:
this is no longer true? The whole thing seems strange (i.e. having CHECK under if DCHECK), but still wondering what's happened

Reply:
From what I recall this was now being called with 0 width/height. I am assuming some layouting change that wasn't happening before. @sangwoo108 thought it was ok to remove, I believe.